### PR TITLE
Use toolchains from the git clone source instead of from the local path

### PR DESCRIPTION
**Pull request by qzheng527**
Signed-off-by: Zheng, Qi <huaiqing.zq@antgroup.com>

### CODE REVIEW

File: tools/docker/Dockerfile.aliyunlinux3 [hunk 1]
```diff
@@ -120,32 +120,27 @@ RUN wget http://www.etallen.com/cpuid/cpuid-20200211.x86_64.tar.gz && \
...
-# Install Occlum Java toolchain (JDK 11)
-ARG JDK11_PATH=/opt/occlum/toolchains/jvm/java-11-openjdk
-COPY --from=alpine /usr/lib/jvm/java-11-openjdk $JDK11_PATH
-RUN rm $JDK11_PATH/lib/security/cacerts
-COPY --from=alpine /etc/ssl/certs/java/cacerts $JDK11_PATH/lib/security/cacerts
-COPY toolchains/java /tmp/java
-WORKDIR /tmp
-RUN cd java && ./install_dragonwell.sh && rm -rf /tmp/java
-ENV PATH="/opt/occlum/toolchains/jvm/bin:$PATH"
+RUN cd golang && ./build.sh && rm -rf /tmp/golang
```
Comment (guzongmin) [inline tools/docker/Dockerfile.aliyunlinux3:142]: Does the golang still work in the docker image?

File: tools/docker/Dockerfile.centos8.2 [hunk 2]
```diff
@@ -162,7 +162,6 @@ RUN git clone -b $OCCLUM_BRANCH https://github.com/occlum/occlum && \
 # To do so, we add the script to ~/.bashrc. We cannot use systemd to run AESM
 # as a "real" service since the pid 1 is not systemd in Docker. So we start
 # up AESM service when an user login with an interative shell.
-COPY docker/start_aesm.sh /opt/occlum/
-RUN mkdir -p /var/run/aesmd && echo '/opt/occlum/start_aesm.sh' >> /root/.bashrc
+RUN mkdir -p /var/run/aesmd && echo 'sudo /opt/occlum/start_aesm.sh' >> /root/.bashrc
```
Comment (guzongmin) [inline tools/docker/Dockerfile.centos8.2:165]: Can you please remove "sudo" in the command?

File: tools/docker/Dockerfile.aliyunlinux3 (hunk 1 above)
Comment (qzheng527) [reply tools/docker/Dockerfile.aliyunlinux3:142]: Yes, it is working

File: tools/docker/Dockerfile.centos8.2 (hunk 2 above)
Comment (qzheng527) [reply tools/docker/Dockerfile.centos8.2]: Done

**Review by guzongmin — APPROVED** [review]
LGTM

**Pull request merged**